torchvision/prototype/transforms/_meta.py

```diff
@@ -42,7 +42,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 class ConvertColorSpace(Transform):
     # F.convert_color_space does NOT handle `_Feature`'s in general
-    _transformed_types = (torch.Tensor, features.Image, PIL.Image.Image)
```

Comment:
Maybe we should have an alias for these three, since they will often happen together?

Reply:
We can do on a follow up. Good idea for when we review the whole input validation mechanism as a whole.